android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageKeystoreAesCbc.java

```diff
@@ -0,0 +1,236 @@
+package com.oblador.keychain.cipherStorage;
```

Comment:
According to Google standard (and its a default for Android Studio IDE and android code), all well-known abbreviations should use a first capital letter instead of all capitals in class names.

Reply:
hello, I'm not objecting it, the trouble I had is that in code review, the old file showed as deleted, and the new files showed as an entirely new file, so it's harder to see the diff.